### ISSUE TITLE
Add github-desktop-dev-auth to list of mimetypes

### ISSUE THREAD

**Issue by Raymo111** [shiftkey/desktop#276]
### Describe the feature or problem you’d like to solve
Upon pushing a change to a workflow, GitHub desktop demanded additional priviledges, but xdg-open wouldn't open the resulting ouath link in desktop and so desktop would fail to push. When I manually added the mimetype to the .desktop file it worked.

### Proposed solution
Add `x-scheme-handler/x-github-desktop-dev-auth;` to the list of mimetypes in the .desktop file.
Original:
```
[Desktop Entry]
Name=GitHub Desktop
Comment=Extend your GitHub workflow beyond your browser with GitHub Desktop
Comment[es]=Trabaja con GitHub desde tu escritorio.
Comment[eu]=GitHub-ekin lan egin zure ordenagailutik.
Exec=/usr/bin/github-desktop %U
Terminal=false
Type=Application
Icon=github-desktop
Categories=Development;
MimeType=x-scheme-handler/x-github-client;x-scheme-handler/x-github-desktop-auth;
```
Updated:
```
[Desktop Entry]
Name=GitHub Desktop
Comment=Extend your GitHub workflow beyond your browser with GitHub Desktop
Comment[es]=Trabaja con GitHub desde tu escritorio.
Comment[eu]=GitHub-ekin lan egin zure ordenagailutik.
Exec=/usr/bin/github-desktop %U
Terminal=false
Type=Application
Icon=github-desktop
Categories=Development;
MimeType=x-scheme-handler/x-github-client;x-scheme-handler/x-github-desktop-auth;x-scheme-handler/x-github-desktop-dev-auth;
```
I would have opened a PR but I couldn't figure out where the .desktop file was (and searching is disabled for forks rip) and also I have a PR to the upstream desktop so I can't fork this repo.

**Comment by shiftkey** [comment]
> I would have opened a PR but I couldn't figure out where the .desktop file was (and searching is disabled for forks rip) and also I have a PR to the upstream desktop so I can't fork this repo.

@Raymo111 can you clarify which installer and version you were testing, as this has changed recently and I want to confirm we're testing the latest version?

**Comment by Raymo111** [comment]
I see. I'm currently using `Version 2.4.0-linux2`.

**Comment by shiftkey** [comment]
@Raymo111 and which installer? `deb`? `AppImage`? `rpm`?

**Comment by Raymo111** [comment]
@shiftkey AUR.

**Comment by Raymo111** [comment]
Which is apparently out of date now
```
aur/github-desktop-bin 2.4.0-1 (+13 2.27%) (Out-of-date 2020-04-29) (Installed)
    GUI for managing Git and GitHub.
```

**Comment by shiftkey** [comment]
Correct. This was added to the `deb` package as part of #263 https://github.com/shiftkey/desktop/pull/263/commits/2325229044b916a6521ed739a5cab35a61f06485

Based on the latest PKGBUILD I believe they use the deb to create the package for Arch: 

https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=github-desktop-bin

Thus, I believe this will be fixed by updating to `2.4.1-linux2`. 

I'm not quite sure when that will happen, but it's not something I have control over.

**Comment by Raymo111** [comment]
@shiftkey Awesome. We'll wait and see. Thanks for maintaining the Linux repo btw, I love GH desktop and I was worried I'd lose it when I switched over to Linux last summer :) Kudos!
